### ISSUE TITLE
Add advanced functionality to handle processing of big datasets

### ISSUE THREAD

**Issue by iretex** [risenW/datasist#55]
Adding pyspark module to be triggered in the event that the size of input dataset is bigger than 1mil rows.
This capability should improve the runtime generally and give improved user experience.

**Comment by risenW** [comment]
Sure, please go ahead

**Comment by iretex** [comment]
Krisblarq, do you want to start writing the test cases, how do you think we can collaborate?
Rising, I imagine that you'll like to retain existing code structure, which could mean that the pyspark code will be maintained as a separate module?
If we agree on the structure, the development can be accelerated.

Very Best!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by risenW** [comment]
Yes @iretex the module should be separate. Then it can be called from other modules when we intend to use it. This should be invisible to the user as well.

**Comment by iretex** [comment]
Very clear!
________________________________
